test/test.js

```diff
@@ -37,4 +37,14 @@ describe('postcss-at2x', () => {
   it('should process image and add background size', (done) => {
     runTest('read-background-size.css', {detectImageSize: true}, done);
   });
+
+  it('should resolve image path with custom function', (done) => {
+    runTest('resolve-image-path.css', {
+      detectImageSize: true,
+      resolveImagePath: (value, source) => {
+        expect(typeof value).toBe('string');
+        expect(source).toHaveProperty('input.css');
+        return path.resolve(process.cwd(), value);
+      }}, done);
+  });
```

Comment:
Yeah, I suppose we are relying on `done` being called on the next tick which is perhaps a bit unusual. I thought of bypassing `runTest` and calling `postcss` manually but this works I guess :)